tests/mutex/LockMutexTest.php

```diff
@@ -92,8 +92,6 @@ public function testUnlockFailsAfterCode()
     /**
      * Tests unlock() fails after the code threw an exception.
      *
-     * The previous exception should be the code's exception.
```

Comment:
Was this not the case or did this change? 

I agree that the previous set up was flawed. 

Just wondering if we broke backwards compatibility.

Reply:
It was not the case. The previous exception is set only in two places and they are (P)RedisExceptions during `eval`.